### ISSUE TITLE
when adding a domain there is no domain validation

### ISSUE THREAD

**Issue by elypter** [Jaxkr/Truthbot.org#17]
you could either use urlparse() or a regex from
http://stackoverflow.com/questions/3026957/how-to-validate-a-domain-name-using-regex-php

**Comment by Jaxkr** [comment]
Yeah, I used to use a regex but it was broken so I temporarily disabled it. https://github.com/Jaxkr/Truthbot.org/commit/7d76b4d68f38cc2b3e75d5a62280dfab5a12e06c

It was failing on long fancy new TLDs like .travel and .museum. Will re-enable soon.
